dynatrace/dynatrace.go

```diff
@@ -121,7 +121,10 @@ func (e *Exporter) Export(ctx context.Context, res *resource.Resource, reader ex
 
 			for iter.Next() {
 				label := iter.Label()
-				dims = append(dims, NewDimension(string(label.Key), label.Value.AsString()))
+				// Drop non-string values
+				if label.Value.Type() == attribute.STRING {
```

Comment:
Should we add some tests to cover this now? Like these https://github.com/dynatrace-oss/opentelemetry-metric-java/blob/main/dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/SerializerTest.java#L63